server/gitlab/api.go

```diff
@@ -373,12 +373,12 @@ func (g *gitlab) GetLabelDetails(client *internGitlab.Client, pid int, labels in
 	var labelsWithDetails []*internGitlab.Label
 	for _, label := range labels {
 		labelWithDetails, resp, err := client.Labels.GetLabel(pid, label)
-		if respErr := checkResponse(resp); respErr != nil {
-			return nil, respErr
-		}
 		if err != nil {
 			return nil, errors.Wrap(err, "can't get label in GitLab api")
 		}
+		if respErr := checkResponse(resp); respErr != nil {
+			return nil, respErr
+		}
```

Comment:
Revert this change. We are checking respErr before err because in respErr we are checking for some specific type of errors from the status code returned in the response.